### DISCUSSION TITLE
Git Clone, Login Git

### DISCUSSION THREAD

**Discussion by MehmetCagferoglu** [irgroup-classrooms/dis08-2021#23] [Q&A]
Hallo, Git fragt mich während "Git clone" nach meinen Login Daten. Ich melde mich dann mit meinen Git Zugangsdaten an aber am ende steht dann die folgende Fehlermeldung.
<img width="793" alt="asdsad" src="https://user-images.githubusercontent.com/49283874/140956926-e130b70c-336f-4be0-acf4-25cefc918447.PNG">

Hat da jemand ein Tipp?

Danke!

**Comment by phschaer** [comment]
Steht eigentlich in der Fehlermeldung drin: "Support for password authentication was removed... " - Details hier: https://github.blog/2020-12-15-token-authentication-requirements-for-git-operations/

Was genau zu tun ist, damit Git und GitHub über die Kommandozeile weiter funktionieren steht hier: https://github.blog/2020-12-15-token-authentication-requirements-for-git-operations/#what-you-need-to-do-today 

Kurzform: Access-Token anlegen und glücklich sein. 

Viel Erfolg!
